.github/workflows/build.yml

```diff
@@ -81,6 +82,13 @@ jobs:
                   build_doc = "false"
           oses = ["macos", "ubuntu", "windows"]
           build_dict = {os : [k for k in build if k.startswith(os)] for os in oses}
+          python_version_per_os = {os: python_version for os in oses}
+          # remove python 3.11 for windows
```

Comment:
What is the issue with Python 3.11 on Windows (could it be added to this comment to not forget in the future and fix when possible)?

Reply:
There is a conflict version on dependencies (I think coming from pyarrow) with python >=3.11. Thus we cannot install scikit-decide[all] and test the windows python3.11 wheel. So I chose not to publish this wheel for now. We could also decide to publish it untested but I am not sure this is really a good idea.

Reply:
I added the reason on the PR description. Is it sufficient for you or do you prefer i put it also in build.yml?

Reply:
Thanks for the explanations, it is fine for me as you did.